### ISSUE TITLE
bakta location table needs a version

### ISSUE THREAD

**Issue by bgruening** [galaxyproject/tools-iuc#4793]
Hi @pimarin,

since bakta versions depend on a specific version of a database, I think bakta needs a version column in https://github.com/galaxyproject/tools-iuc/blob/master/tools/bakta/tool_data_table_conf.xml.sample#L4

This enables different bakta Galaxy tool versions to take the same location file, but filters for the DB version. Does this make sense?

Cheers,
Bjoern

**Comment by pimarin** [comment]
Hi @bgruening,
You right, but how it work ? Just add a version column ?

**Comment by pimarin** [comment]
> Yes, a version column and then filter in the tool for this column and only show versions that are allowed.

Okay so now : `  <columns>value, name, version, path</columns>` with the version of the DB then a filter option in bakta.xml like : `<filter type="static_value" column="column_number_of_bakta_version" value="@TOOL_VERSION@" />`

**Comment by bgruening** [comment]
Yes kindoff. The DB version does not correspond to the tool version imho

**Comment by bgruening** [comment]
The star DM and tool uses versioned location files.
